src/Swarm/TUI/Model.hs

```diff
@@ -258,6 +265,15 @@ uiFPS :: Lens' UIState Double
 -- | The base-2 logarithm of the current game speed in ticks per
 --   second.
 lgTicksPerSecond :: Lens' UIState Int
+lgTicksPerSecond = lens _lgTicksPerSecond safeSetLgTicks
+ where
+  maxTicks = 5 -- 2^(-5) == 1/32
```

Comment:
The `ticksPerFrameCap` is ticks per *frame*, not ticks per second.  There is no reason to limit the max number of ticks per second to 32.  I know my computer can easily handle higher than that depending on how many robots there are.